src/repository/entity/OwnerEntity.java

```diff
@@ -0,0 +1,48 @@
+package repository.entity;
+
+import java.time.LocalDate;
+import java.util.UUID;
+public class OwnerEntity extends BaseEntity{
```

Comment:
Add blank line after import pls
To reformat code easily, make sure you use ctrl-alt-L combination